### ISSUE TITLE
[MU4 Issue] [Accessibility] VoiceOver doesn't read notes in score

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9578]
VoiceOver reads "group" instead of appropriate note - see video below

https://user-images.githubusercontent.com/90187801/138893072-7d8835b3-4fd3-4708-bf42-bb5ccaf7fcde.mov

**Desktop:**
 Occurs only with VoiceOver (Mac11.6)

**Comment by DmitryArefiev** [comment]
Rechecked on latest master build, Mac11.6 - **STILL OCCURS**
